src/lib/participants/registration.ts

```diff
@@ -1,6 +1,6 @@
-export const registrationOpensAt = +new Date('2023-05-01 00:00:00Z');
-export const eventStartsAt = +new Date('2023-06-30 09:30:00Z');
-export const registrationClosessAt = +new Date('2023-06-26 17:00:00Z');
+export const registrationOpensAt = +new Date('2024-05-01 00:00:00Z');
+export const eventStartsAt = +new Date('2024-06-30 09:30:00Z');
+export const registrationClosessAt = +new Date('2023-04-26 17:00:00Z');
```

Comment:
That's... interesting...! I don't know whether I tested the case of having registration close before it opens... 😬

Reply:
not relevant yet... we should change it once we know